src/components/CarouselImage/index.js

```diff
@@ -127,38 +134,43 @@ class Item extends Component {
         const { assistiveText, description, header, href, style } = this.props;
         const hasContent = !!(header || description);
```

Comment:
this logic belongs to ImageContainer, since already you are passing header and description to it.